src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  height: 408px;
+  background: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  padding-top: 32px;
+  padding-inline: 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    width: 166px;
+    height: 36px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-top: 40px;
+    color: $main-text-color;
```

Comment:
Ensure that the variable `$main-text-color` is defined in your SCSS files. If it's not defined, this will cause a compilation error.